README.md

```diff
@@ -744,6 +744,10 @@ parameters:
          - /go/bin/pmem-ns-init
 ```
 
+#### Note about raw block volumes
+
+Applications can use volumes provisioned by PMEM-CSI as [raw block devices](https://kubernetes.io/blog/2019/03/07/raw-block-volume-support-to-beta/). For provisioning a PMEM volume that supposed to use as raw block device, one has to create a `PersistentVolumeClaim` with `volumeMode: Block`. Check with provided example [PVC](deploy/common/pmem-pvc-block-volume.yaml) and [application](deploy/common/pmem-app-block-volume.yaml) for usage reference.
```

Comment:
instead of:
`For provisioning a PMEM volume that supposed to use as raw block device`
how about simpler:
`For provisioning a PMEM volume as a raw block device`

similarly:
instead of:
`Check with provided example` 
use:
`See example`

We can also wait for few more days when @pohly gets back, he can also take a look on these sentences.

Reply:
I agree with @okartau's proposed language changes.

Reply:
Those changes are al

> Olev Kartau <notifications@github.com> writes:
> > Do we already have tests (e.g. from Kubernetes or csi-test) which cover block volumes? while working with #365 I noticed that if we dont respond NOT_SUPPORTED for block volume mode, and we enable VolumeMode suite, then some (just few) more tests are executed. I guess these may be about block volumes.
> Can we pull that into this PR?

Now I pulled the [commit](https://github.com/intel/pmem-csi/pull/365/commits/0875b6dc22ccf69414266876df8e5e30de798e46) from #365 that enables the block volume test suite.